src/uct/ib/mlx5/dc/dc_mlx5_ep.h

```diff
@@ -368,8 +368,7 @@ uct_dc_mlx5_ep_basic_init(uct_dc_mlx5_iface_t *iface, uct_dc_mlx5_ep_t *ep)
         }
         ep->dci               = 0;
         dci                   = uct_dc_mlx5_iface_dci(iface, ep->dci);
-        ep->dci_channel_index = dci->next_channel_index++ %
-                                iface->tx.num_dci_channels;
+        ep->dci_channel_index = dci->next_channel_index++;
```

Comment:
Can't it grow infinitely?

Reply:
dci_channel_index is uint8_t so it will go back to 0 after 255

Reply:
how is it handled in HW? Do they do module by channel num themselves?